digits/job.py

```diff
@@ -202,3 +211,44 @@ def disk_size_fmt(self):
         size = fs.get_tree_size(self._dir)
         return sizeof_fmt(size)
 
+    def get_tasks_recursively(self):
+        tasks = []
+        for task in self.tasks:
+            if task:
+                tasks.append(task)
+                tasks += task.get_tasks_recursively()
+        return tasks
```

Comment:
No need to change it at this point.  I was just working with certain assumptions about the graph.  I removed that method in job.py and task.py.  So, I'm happy with that.